### ISSUE TITLE
US #3 - Ambient Background

### ISSUE THREAD

**Issue by SamvritSrinath** [cse110-sp24-group7/warmup-exercise#4]
## Description

Change the background color based on where the user hovers over the webpage —> this will be abstracted to when the user goes over a given portion of the rainbow, the webpage changes. 

## Frontend

See Task Description —> try to use mood-facing colors that reflect a given mood(at a 7-point scale). 

## Backend

N/A

**Comment by arnavkamdar** [comment]
My assumption was that the background changes based on what we're hovering over in the widget, not the webpage. @SamvritSrinath what do you think?

**Comment by SamvritSrinath** [comment]
> My assumption was that the background changes based on what we're hovering over in the widget, not the webpage. @SamvritSrinath what do you think?

This is the intended purpose, so let me rephrase the task(I'll also update this above)
**The goal is to be able for the color of the widget/rainbow to dynamically change as you go along the rainbow**, however I think it also will be cool as a design element to have the color of the webpage change **gradients** as a user moves their cursor along the page for User Interactivity(maybe an @media element or something else) —> our current wireframe has us just display quotes on a webpage, so it might be cool for user experience.

**Comment by SamvritSrinath** [comment]
Tasked to @hfituri
